src/utils/filterPeople.ts

```diff
@@ -0,0 +1,18 @@
+import { Person } from '../types/Person';
+
+export function filterPeople(
+  people: Person[],
+  { appliedQuery }: { appliedQuery: string },
+) {
+  let filteredPeople = [...people];
+
+  if (appliedQuery) {
```

Comment:
You should check if `appliedQuery` is not only truthy, but also not just spaces. Currently, if a user enters spaces only, the filter function will be called unnecessarily. Consider using `trim()` method before the check to ensure that spaces only won't pass the condition.